cura/UI/WelcomePagesModel.py

```diff
@@ -7,6 +7,7 @@
 
 from UM.Qt.ListModel import ListModel
 from UM.Resources import Resources
+from logging import Logger
```

Comment:
Seems like an unused import from the diff in this PR :laughing: 
```suggestion
```

Reply:
Removed.